pcdscalc/be_lens_calcs.py

```diff
@@ -1,14 +1,17 @@
 """Module for Beryllium Lens Calculations."""
+import json
 import logging
 import os
 import shutil
 from datetime import date
-from itertools import product
+from itertools import chain, product
 
 import numpy as np
 import xraydb as xdb
 
 logger = logging.getLogger(__name__)
+# global variable to be help with testing the plan_set function
+_plan_set_test_res = None
```

Comment:
I think this was a good solution and doesn't need a revision. Another way to do this is to have testable helper functions with whatever return values we want, and use the helpers in the functions without good return values.